src/lib/save_result.ml

```diff
@@ -63,7 +63,7 @@ let make_saving_node
       )
   in
   let name =
-    sprintf "Salvation: %s" (Filename.basename path) in
+    sprintf "Saved: %s" key in
```

Comment:
"Copied to final"?

On Tue, Sep 13, 2016 at 4:11 PM Sebastien Mondet <notifications@github.com>
wrote:

> In src/lib/save_result.ml
> <https://github.com/hammerlab/epidisco/pull/26#discussion_r78634769>:
>
> > @@ -63,7 +63,7 @@ let make_saving_node
> >        )
> >    in
> >    let name =
> > -    sprintf "Salvation: %s" (Filename.basename path) in
> > +    sprintf "Saved: %s" key in
>
> @ihodes <https://github.com/ihodes> @arahuja <https://github.com/arahuja>
> please agree on a name :)
> I'm doing another PR (to save Seq2HLA)
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/hammerlab/epidisco/pull/26/files/4c8d2191e2f3157ed498e44141b569444b3edeeb#r78634769>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAKPe_cPXU44jtjOohXeiGz4shZFAPyrks5qpwN_gaJpZM4J8EQl>
> .
>